packages/ironfish-native-module/src/index.ts

```diff
@@ -77,3 +77,21 @@ export function decryptNotesForOwner(
     incomingHexKey,
   );
 }
+
+export function decryptNotesForSpender(
+  noteEncrypted: string[],
+  outgoingHexKey: string,
+): Promise<{ index: number; note: string }[]> {
+  return IronfishNativeModule.decryptNotesForSpender(
+    noteEncrypted,
+    outgoingHexKey,
+  );
+}
+
+export function nullifier(
```

Comment:
suggestion (low priority): What do you think about making these functions take an object as args so we don't have to rely on argument order?

Reply:
Sure, do you think it'd be okay to parse all the fields off the object and send in the arguments directly to the function? It takes a bit of boilerplate to model argument-objects in the Swift/Kotlin bindings. Basically like this:

```ts
export function nullifier({
  note,
  position,
  viewKey,
}: {
  note: string;
  position: string;
  viewKey: string;
}): Promise<string> {
  return IronfishNativeModule.nullifier(note, position, viewKey);
}
```

Reply:
Yeah, definitely